src/components/diagrams/diagram-pane.js

```diff
@@ -1171,5 +1175,6 @@ DiagramPane.propTypes = {
     showInSpreadsheet: PropTypes.func,
     isComputationRunning: PropTypes.bool,
     loadFlowStatus: PropTypes.any,
+    oneBusShortCircuitStatus: PropTypes.any,
```

Comment:
it have to be a string PropTypes.string and I think that it must be required otherwise it must have a default value.

Reply:
Done